lib/DDG/Spice/WGHA.pm

```diff
@@ -20,13 +20,12 @@ my @triggers = ("was geht", "wasgeht", "aus gehen", "ausgehen", "nachtleben", "p
 triggers any => @triggers;
 
 spice to => 'http://ddg.wasgehtheuteab.de/duckduckgo/events.json?q=$1&callback={{callback}}&api_key={{ENV{DDG_SPICE_WGHA_APIKEY}}}';
-spice proxy_cache_valid => "200 60m";
+spice proxy_cache_valid => "200 5m";
 
 handle query_lc => sub {
-  return $_ if $loc->country_code eq 'DE';
-
-  if ($_ =~ /aschaffenburg|augsburg|berlin|bremen|darmstadt|dortmund|dresden|düsseldorf|duesseldorf|essen|frankfurt|ffm|freiburg|hamburg|hannover|heidelberg|kassel|köln|koeln|leipzig|mainz|mannheim|münchen|muenchen|münster|muenster|nürnberg|nuernberg|stuttgart|wiesbaden/) {
-    return $_;
+  $a = $_ . ' ' . $loc->city;
+  if ($a =~ /aschaffenburg|augsburg|berlin|berlin-biesdorf|berlin-friedrichshain|berlin-karow|berlin-köpenick|berlin-lichtenberg|berlin-marzahn|berlin-mitte|berlin-oberschöneweide|berlin-pankow|berlin-treptow|bremen|cologne|darmstadt|dieburg|dortmund|dreieich|dresden|dresden-altstadt|dresden-neustadt|düsseldorf|essen|ffm|frankfurt|frankfurt am main|freiburg|freiburg im breisgau|friedrichshain|griesheim|halle|hamburg|hamburg-eimsbüttel|hamburg-mitte|hannover|hanover|heidelberg|kassel|koeln|germany|kreuzberg|köln|leipzig|leipzig-connewitz|leipzig-leutzsch|leipzig-plagwitz|leipzig-stötteritz|mainz|mannheim|munich|mühlheim|münchen|münchen|germany|münster|neu isenburg|nuremberg|nürnberg|offenbach|prenzlauer berg|sankt pauli|sersheim|stuttgart|wiesbaden/) {
```

Comment:
In the interest of saving more CPU cycles :), I think we should put this list into a JSON/YAML file so we can load it into a hash and do a hash lookup instead of using a regex. Alternatively, we could build the regex outside the `handle` function so we don't need to re-build the hash each time the function is executed?